docu/files/ModuleExtensibility.html.md

```diff
@@ -31,15 +30,15 @@ Available templates:
 4. uiExtensibilityPlugin - Plug-in module example
 ```
 
-When you view the generated files, you will see that they exhibit some new features compared to a simple Builder module:
+Notice that the generated files exhibit some new features compared to a simple Builder module:
```

Comment:
The generated files exhibit some new features compared to a simple Builder module: